examples/worker-pools/worker-pools.go

```diff
@@ -25,24 +25,28 @@ func main() {
     // In order to use our pool of workers we need to send
     // them work and collect their results. We make 2
     // channels for this.
-    jobs := make(chan int, 100)
-    results := make(chan int, 100)
+    jobs := make(chan int, 2)
```

Comment:
this change is to highlight the reader the reason for having a separate go routine for setting up the job queue.